src/core/device.c

```diff
@@ -202,6 +202,13 @@ static void device_dump(Unit *u, FILE *f, const char *prefix) {
                 prefix, strna(d->sysfs));
 }
 
+_pure_ static int device_reload(Unit *u) {
```

Comment:
"pure" doesn appear right for this. This function returns a constant "0" anyway...